src/shared/components/Header/HomeHeader/HomeHeader.css.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+export const HeaderWrapper = style({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  padding: '1.1rem 2.4rem',
+  width: '375px',
+  boxSizing: 'border-box',
+  backgroundColor: ' #FFF',
```

Comment:
```suggestion
import { vars } from '@/shared/styles/tokens.css'; 



backgroundColor: vars.color.white 
```

피그마에 있는 색상 코드 토큰으로 정의해두었으니 색상 코드들 vars import하고 위처럼 변경 부탁드립니다 !!!

Reply:
넹! 수정했습니당! (❁´◡`❁)